nixos/modules/services/torrent/transmission.nix

```diff
@@ -86,6 +86,8 @@ in
         ExecStartPre = ''
           ${pkgs.stdenv.shell} -c "chmod 770 ${homeDir} && mkdir -p ${settingsDir} ${downloadDir} ${incompleteDir} && rm -f ${settingsDir}/settings.json &&   cp -f ${settingsFile} ${settingsDir}/settings.json"
         '';
+        # allow to bind on ports below 1024
+        AmbientCapabilities = "cap_net_bind_service";
```

Comment:
I think it makes sense to make this conditional on (`cfg.ports < 1024`) and I would also like to see a warning to the user that the daemon will run with elevated capabilities.